internal/cmd/release/main.go

```diff
@@ -182,10 +181,6 @@ func (c *command) calculateNewReleasePlugins(currentRelease *release.PluginRelea
 	var existingPlugins []release.PluginRelease
 
 	if err := plugin.Walk(c.rootDir, func(plugin *plugin.Plugin) error {
-		identity, err := bufpluginref.PluginIdentityForString(plugin.Name)
```

Comment:
There were several places where we converted a plugin name to an identity. I updated this to happen when we load plugins so we don't have to do it (and handle errors) everywhere.